ChangeLog.md

```diff
@@ -4,7 +4,9 @@
   - Upgrade of ITK to version 5 and above (Bastien Doignies, [#1757](https://github.com/DGtal-team/DGtal/pull/1757))
   - C++20 support and new minimal standard (Bastien Doignies, [#1757](https://github.com/DGtal-team/DGtal/pull/1757))
   - Library is now (almost) header only (Bastien Doignies, [#1757](https://github.com/DGtal-team/DGtal/pull/1757))
-  - Fix compilation warnings (Bastien Doignies, [#1757](https://github.com/DGtal-team/DGtal/pull/1757))
+  - Fix compilation warnings (Bastien Doignies, [#1757](https://github.com/DGtal-team/DGtal/pull/))
+  - Remove GMP dependancy (Bastien Doignies, [#1769](https://github.com/DGtal-team/DGtal/pull/1769))
```

Comment:
```suggestion
  - Remove GMP dependency (Bastien Doignies, [#1769](https://github.com/DGtal-team/DGtal/pull/1769))
```

Reply:
Thanks, I haven't seen it. Done : here https://github.com/DGtal-team/DGtal/pull/1769/commits/46e2cfd1651dd9408f3a547fdca222b4e44565ad#r2009664955